src/utils.jl

```diff
@@ -161,3 +161,57 @@ returns a boolean flag to signal a successful operation.
 """
 check_errors(success::Integer, doc::VPtr, error::Exception) =
     Bool(success) || get_error_messages(doc, error)
+
+# NOTE: this mapping is valid for Level 3/Version 2, it *may* not be valid for
+# other versions
```

Comment:
I was wondering if we wouldn't be "safer" with using SBML to convert the unit ID to string (unit string names are imho more stable than enums) and only map from Strings to Units here. I had that with the original units (ccall `UnitKind_toString`).

Reply:
Done!